Configure

```diff
@@ -233,6 +233,7 @@ my @dtls = qw(dtls1 dtls1_2);
 
 my @disablables = (
     "aes",
+    "afalg",
```

Comment:
You probably should change that to `"afalgeng"`, so it gets translated properly to `OPENSSL_NO_AFALGENG`

Also, that would be consistent, since we already have `"capieng"`